packages/browser/src/Widget/Server/Workspace/Readme.md

```diff
@@ -0,0 +1,21 @@
+# Workspace Module
+
+The Workspace Module is an extension of the SPARQL Module that adds functionality to query and visualize Workspace, and CityGML Versioning data stored in a Strabon Triple-store from a UD-Viz interface.
+
+See the [SPARQL module](../SPARQL/) for more documentation on generic SPARQL functions
+
+## Workspace functionalities
+This module extends the functionality of the [SPARQL module](../SPARQL/) with the ability to update the displayed 3D model by clicking on **Version** and **VersionTransition**. These nodes are displayed in a D3 graph created from Workspace and Versioning graph data from a SPARQL backend service.
+
+If a **VersionTransition** is selected  and highlight changes between the related versions of the 3D city model will be highlighted.
```

Comment:
maybe specify that 3DTiles and sparql database have to be sync in a way to works correctly (data in both have to be sync)